scripts/templates/locals.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_organizations_organization" "root_account" {}
+data "aws_caller_identity" "current" {}
+
+locals {
+
+  team_name = "#TEAM#"
+  suffix    = "#SUFFIX#"
+
+  root_account           = data.aws_organizations_organization.root_account
```

Comment:
These declarations of root_account needed (and the dependent data.aws_organization.root_account)?